index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <div class="status-bar">
+        <div class="clock">9:40</div>
+        <div class="status"><img src="Img/status.png" alt="wifi"></div>
+        <div class="name">Arturo Hermida</div>
+        <div class="dropdown"><img src="Img/Enabled.png" alt="drop"></div>
+    </div>
+    <div class="content">
+        <h1 class="hello">Hey there.<br>I'm Arturo.</h1>
+        <h3 class="profession">I'am a Software Developer</h3>
+        <p class="description">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don't hestiate to contact me.</p>
+        <div class="social">         
+            <a href="#" class="icon1"><img src="Img/github.png" alt="github"></a>
+            <a href="#" class="icon2"><img src="Img/linkedin.png" alt="linkedin"></a>
+            <a href="#" class="icon3"><img src="Img/icon3.png" alt="icon3"></a>
+            <a href="#" class="icon4"><img src="Img/twitter.png" alt="twitter"></a>
+            <a href="#" class="icon5"><img src="Img/icon5.png" alt="icon 5"></a>
+    </div>
+    <img src="Img/design.png" alt="design" class="ilustration">
+    </div>
```

Comment:
- [ ] This image is not a part of the content, therefore please make sure to load it as a background image from `css`.